doc/dev/index.rst

```diff
@@ -178,12 +178,75 @@ with:
     # check that it's there
     ./ceph health
 
+Architecture
+============
+
+Ceph is a collection of components built on top of RADOS and provide
+services (RBD, RGW, CephFS) and APIs (S3, Swift, POSIX) for the user to
+store and retrieve data.
+
+See :doc:`/architecture` for an overview of Ceph architecture. The
+following sections treat each of the major architectural components
+in more detail, with links to code and tests.
+
+.. FIXME The following are just stubs. These need to be developed into
+   detailed descriptions of the various high-level components (RADOS, RGW,
+   etc.) with breakdowns of their respective subcomponents.
+
+.. FIXME Later, in the Testing chapter I would like to take another look
+   at these components/subcomponents with a focus on how they are tested.
+
+RADOS
+-----
+
+RADOS stands for "Reliable, Autonomic Distributed Object Store". In a Ceph
+cluster, all data are stored in objects, and RADOS is the component responsible
+for that.
+
+RADOS itself can be further broken down into Monitors, Object Storage Daemons
+(OSDs), and client APIs (librados). Monitors and OSDs are introduced at
+:doc:`/start/intro`. The client library is explained at
+:doc:`/rados/api/index`.
+
+RGW
+---
+
+RGW stands for RADOS Gateway. Using the embedded HTTP server civetweb_, RGW
```

Comment:
or using Apache FastCGI? (I'm not sure whether we've deprecated fcgi for jewel yet)